spec/system/support_interface/validation_errors_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.feature 'Validation errors' do
+  include DfESignInHelpers
+
+  scenario 'Review validation errors' do
+    given_i_am_a_support_user
+    and_there_are_some_validation_errors
+
+    when_i_navigate_to_the_validation_errors_page
+    then_i_should_see_a_list_of_error_groups
+
+    when_i_click_on_a_group
+    then_i_should_see_a_list_of_individual_errors
+
+    when_i_click_on_link_in_breadcrumb_trail
+    then_i_should_be_back_on_index_page
+  end
+
+  def given_i_am_a_support_user
+    sign_in_as_support_user
+  end
+
+  def and_there_are_some_validation_errors
+    @candidate = create(:candidate, email_address: 'bob@example.com')
+    @validation_error = create(
+      :validation_error,
+      form_object: 'CandidateInterface::DegreeForm',
+      details: {
+        award_year: { value: '2222', messages: ['Enter a year before 2022'] },
+      },
+      request_path: '/candidate/application/degrees/51/year',
+      created_at: Time.zone.local(2020, 4, 24, 12, 35, 42),
+      user: @candidate,
+    )
```

Comment:
We could consider going into the signup form and trigger an error for the full end-to-end experience.